_sass/_hero.scss

```diff
@@ -19,13 +19,13 @@ $hero--card-count: $hero--columns * $hero--rows;
     display: grid;
     place-content: center;
 
-    font-size: 2.5rem;
+    font-size: clamp(2.5rem, calc(5vw + 0.5rem), 4rem);
```

Comment:
I think the maximum font size for the <h1> should be 3rem. It aligns better with the width of the central column in the grid (and at some point if it's too big it is a little loud.)

Learned something with `clamp`, looks like a great way to save on media query usage.

Reply:
Also a touch more for the line-height!